caom2persistence/README.md

```diff
@@ -1,3 +1,9 @@
 # caom2db/caom2persistence
 
 If a file named 'test.schema' is available in this directory, the contents of that file will be used as the schema for running the unit tests.
+
```

Comment:
The caom2persistence library is designed to update itself to the current version and tracks the current version in caom2.ModelVersion table. I don't see why one would want to disable that - it is an idempotent operation and if it is skipped or fails then other stuff is also going to fail (because the version in InitDatabase is only set for those versions that change the DDLs). Why would you want to skip?

Reply:
So this is to guarantee that the latest DDL is being used for the test?  It doesn't handle an existing CAOM-2 database though.